src/app/app.component.html

```diff
@@ -2,7 +2,7 @@
     <hc-navbar-link uri="/home" linkText="Home"></hc-navbar-link>
     <hc-navbar-link uri="/styles" linkText="Styles"></hc-navbar-link>
     <hc-navbar-link uri="/components" linkText="Components"></hc-navbar-link>
-    <hc-navbar-link uri="/guides/getting-started" linkText="Guides"></hc-navbar-link>
```

Comment:
This was causing a routing bug when navigating to any guides other than getting started.  The navbar didn't highlight guides as the active route.